plugins/ui-converter-plugin/src/modules/Server/UIConverterUtils.lua

```diff
@@ -199,15 +198,15 @@ function UIConverterUtils.toLuaPropertyString(value, debugHint)
 		end
 	elseif valueType == "Font" then
 		if value.Weight == Enum.FontWeight.Regular and value.Style == Enum.FontStyle.Normal then
-			return ("Font.new(%q)"):format(value.Family)
+			return string.format("Font.new(%q)", value.Family)
 		else
-			return ("Font.new(%q, %s, %s)"):format(value.Family, tostring(value.Weight), tostring(value.Style))
+			return string.formatstring.format("Font.new(%q, %s, %s)", value.Family, tostring(value.Weight, value.Style))
```

Comment:
check this

Reply:
oh wow I'm surprised selene didn't catch that
fixed in [c4253b0](https://github.com/Quenty/NevermoreEngine/pull/497/commits/c4253b0fa5841f436eed25e1a0c9d0083dd06ca8)